_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -106,11 +100,13 @@ When using segment replication, consider the following:
 
 1. Enabling segment replication for an existing index requires [reindexing](https://github.com/opensearch-project/OpenSearch/issues/3685).
 1. [Cross-cluster replication](https://github.com/opensearch-project/OpenSearch/issues/4090) does not currently use segment replication to copy between clusters.
+1. Segment replication is not compatible with [document-level monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/api/#document-level-monitors), which are used with the [Alerting]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/) plugin. The Alerting plugin uses the latest available data on replica shards when using the `immediate` refresh policy, and segment replication can delay the policy's availability, resulting in stale replica shards.
+1. Segment replication is not compatible with [security analytics]({{site.url}}{{site.baseurl}}/security-analytics/index/). Security analytics uses document-level monitors, which are not compatible with segment replication.  
```

Comment:
Is this duplicate?

Reply:
Revised to combine into one bullet point.